### PR TITLE
映射文档 No.243-300 其他未领取项

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/PaConvert/issues/112

转换规则 https://github.com/PaddlePaddle/PaConvert/pull/217

243 torch.linalg.inv_ex 功能缺失
246 torch.nn.LazyBatchNorm1d 功能缺失
247 torch.Tensor.sparse_resize_and_clear_ 功能缺失
248 torch.linalg.svdvals
251 torch.linalg.matrix_norm 功能缺失
254 torch.gradient 功能缺失
255 torch.special.gammaln
257 torch.nn.utils.parametrize.is_parametrized 功能缺失
259 torch.special.erfc 组合替代实现
260 torch.linalg.vector_norm 功能缺失
262 torch.nn.functional.group_norm 功能缺失
265 torch.special.expit 组合替代实现
266 torch.nn.Mish 已有文档
268 torch.linalg.cholesky_ex 功能缺失
269 torch.Tensor.sparse_resize_ 功能缺失
270 torch.positive 功能缺失
275 torch.special.exp2 组合替代实现
276 torch.nn.utils.parametrizations.spectral_norm
277 torch.cuda.StreamContext 功能缺失
278 torch.nn.LazyBatchNorm2d 功能缺失
281 torch.autograd.profiler.profile.self_cpu_time_total 功能缺失
284 torch.profiler.ProfilerActivity 功能缺失
286 torch.special.entr 功能缺失
287 torch.nn.LazyBatchNorm3d 功能缺失
288 torch.autograd.function.FunctionCtx.mark_dirty 功能缺失
289 torch.autograd.function.FunctionCtx.mark_non_differentiable
290 torch.autograd.Function.forward
291 torch.autograd.function.Function  名称应为 torch.autograd.Function https://pytorch.org/docs/stable/autograd.html#torch.autograd.Function
292 torch.autograd.function.FunctionCtx.set_materialize_grads
293 torch.autograd.function.FunctionCtx.save_for_backward
294 torch.autograd.Function.backward
295 torch.profiler.ProfilerAction 功能缺失
296 torch.cuda.set_stream
300 torch.autograd.graph.saved_tensors_hooks

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### [torch.nn.functional.group_norm](https://pytorch.org/docs/stable/generated/torch.nn.functional.group_norm.html#torch.nn.functional.group_norm)
+
+```python
+torch.nn.functional.group_norm(input, num_groups, weight=None, bias=None, eps=1e-05)
+```
+
+### [paddle.static.nn.group_norm](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/static/nn/group_norm_cn.html#group-norm)
+
+```python
+paddle.static.nn.group_norm(input, groups, epsilon=1e-05, param_attr=None, bias_attr=None, act=None, data_layout='NCHW', name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md]: 这个是静态图API的，这个对不上因为是把Pytorch动态图转换为Paddle动态图

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| check_errors | -            | 是否检查 infos 的内容，Paddle 无此参数，暂无转写方式。                                                                                              |
+| out          | -            | 表示输出的 Tensor ，Paddle 无此参数，需要进行转写。                                                                                                 |
+
+### 转写示例
+
+#### out: 输出的 Tensor
+
+```python
+# Pytorch 写法
+torch.linalg.cholesky_ex(x, upper=False, out=output)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md]: 这两个API名字不太一致，对比了结果是一致的对吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.autograd.Function.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+torch.autograd.Function(*args, **kwargs)
+```
+
+### [paddle.autograd.PyLayer](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/autograd/PyLayer_cn.html#paddle.autograd.PyLayer)
+
+```python
+paddle.autograd.PyLayer()
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.autograd.Function.md]: 参考下 torch.optim.Optimizer 的写法
可写 但该类一般用于继承实现，不会调用其参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+## [ 参数完全一致 ]torch.nn.utils.parametrizations.spectral_norm
+
+### [torch.nn.utils.parametrizations.spectral_norm](https://pytorch.org/docs/stable/generated/torch.nn.utils.parametrizations.spectral_norm.html#torch.nn.utils.parametrizations.spectral_norm)
+
+```python
+torch.nn.utils.parametrizations.spectral_norm(module, name='weight', n_power_iterations=1, eps=1e-12, dim=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md:6]: 这个是不是可以有更简短的调用方式

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+## [ 参数完全一致 ]torch.nn.utils.parametrizations.spectral_norm
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md]: 下面有 仅参数名不一致 了，就不会是这一类

**Review by zhwesky2010 — COMMENTED** [review]
有些组合规则挺复杂的，因此建议与 转换规则 共同实现，互相校正

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md (hunk 2 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md]: torch.linalg.cholesky_ex 是 torch.linalg.cholesky 扩展版本，返回数据增加info，增加参数 check_errors 忽略错误消息
如果需要返回参数一致，那是功能缺失

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md (hunk 1 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md]: 那是功能缺失吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md (hunk 4 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md:6]: https://pytorch.org/docs/stable/generated/torch.nn.utils.spectral_norm.html?highlight=spectral_norm#torch.nn.utils.spectral_norm

torch.nn.utils.spectral_norm 是未来废弃的版本
图片

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.autograd.Function.md (hunk 3 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.autograd.Function.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md (hunk 5 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.utils.parametrizations.spectral_norm.md]: 已修改

**Comment by co63oc** [comment]
> 有些组合规则挺复杂的，因此建议与 转换规则 共同实现，互相校正

转换规则PR https://github.com/PaddlePaddle/PaConvert/pull/217

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md (hunk 1 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md]: 目前确实没有这个API，算功能缺失吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md (hunk 2 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md]: 看起来只能功能缺失了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+| check_errors | -            | 是否检查 info 的内容，Paddle 无此参数，暂无转写方式。 |
+| out          | -            | 表示输出的 Tensor ， Paddle 无此参数，需要转写。  |
+
+### 转写示例
+
+#### out 参数：输出的 Tensor
+
+```python
+# PyTorch 写法:
+torch.linalg.inv_ex(x, out=y)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md]: 这个也会返回info吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+### 参数映射
+
+| PyTorch | PaddlePaddle  | 备注                                                                                 |
+| ------- | ------------- | ------------------------------------------------------------------------------------ |
+| A       | x             | 输入 Tensor，仅参数名不一致。                                                        |
+| driver  | -             | cuSOLVER 方法名，Paddle 无此参数，暂无转写方式。                                     |
+| -       | full_matrices | 是否计算完整的 U 和 V 矩阵，Paddle 为 False，Pytorch 无此参数，Paddle 使用默认即可。 |
+| out     | -             | 表示输出的 Tensor，Paddle 无此参数，需要转写。                                   |
+| 返回值  | 返回值        | PyTorch 返回值为 S，Paddle 返回多个值，需要转写。                                |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md]: 可以写下paddle返回的这三个值

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md (hunk 6 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md]: 图片
返回info

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md (hunk 7 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md (hunk 6 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.inv_ex.md]: 已修改为功能缺失

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md (hunk 2 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.cholesky_ex.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md (hunk 1 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.group_norm.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle  | 备注                                                                                 |
+| ------- | ------------- | ------------------------------------------------------------------------------------ |
+| A       | x             | 输入 Tensor，仅参数名不一致。                                                        |
+| driver  | -             | cuSOLVER 方法名，Paddle 无此参数，暂无转写方式。                                     |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md]: 这个影响计算结果吗，不影响的话就是 `可直接删除` 的话术，可以查看其他API描述直接删除的话术

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svdvals.md]: 计算结果使用的不同算法，可能精度上有较小差异，对结果影响不大，已修改 可直接删除

**Review by zhwesky2010 — APPROVED** [review]
LGTM
